plover/assets/main.json

```diff
@@ -25411,7 +25414,10 @@
 "HOPBLG/KEUPB/AOES": "Hodgkin's",
 "HOPBLG/POPBLG": "hodgepodge",
 "HOPBLG/SKWRE/TPHAEUT": "homogeneity",
+"HOPBLS": "honestly",
+"HOPBS": "honest",
```

Comment:
`HO*PBS` is already defined as `honesty`. I could define `HO*PBLS` → `honestly`, that's not easy to stroke however. So I'll just remove these entries.